api/openapi.yaml

```diff
@@ -502,7 +502,7 @@ paths:
               EIDAS QEC Example:
                 value:
                   did: did:key:zQ3shtxV1FrJfhqE1dvxYRcCknWNjHc3c5X1y3ZSoPDi2aur2
-                  p12: <buffer-input-of-your-certificate>
+                  p12: 308210f9020103308210bf06092a864886f70d010701a08210b0048210ac308210a8308206d706092a864886f70d010706a08206c8308206c...
```

Comment:
May we provide a proper key so the Try out succeeds?

Reply:
Other two requests are ready to be triggered. It'd be nice this last too.

Reply:
🤔  not sure about it. mainly because it requires a password (on the sign call), that we do not explicit tell, so, other devs cannot really test the flow via swagger.

Reply:
> I've checked dependencies pending to update and there are a bunch. I suggest to create another task and PR to upgrade them all. Anyhow, we should upgrade axios to fix high severity vulnerability.
> 
> Aside from that, all tests passed and no lint errors were found 🥇

Regarding library updates, I did not check it. So, i'll update them, even jose... 😱  😄

Reply:
Actually I had to set this passwords so the key could be used through Swagger. Since the key is only for testing purposes @xvilapueyo agreed to expose its password so the requests could work. 

I've just checked this in your openapi.yml and you do not have it. May you have forgotten to update development before creating this branch?